README.md

```diff
@@ -1,37 +1,94 @@
-# Cuddly App
+# Cuddly - Sharing the joy of pets
+
+Cuddly is a social media platform for everything pet related made by [Decio Candido](https://github.com/deciocandido), [En-Chi Liu](https://github.com/eeels22), [Janko Radakovic](https://github.com/janko101), [Pei Nen Esther Chee](https://github.com/estherchee) and [Yash Seeta](https://github.com/NeuralAlchemist).
+
+## Contents
+
+- [Vision](#vision)
+- [Features](#features)
+- [Technologies](#technologies)
+- [Setup](#setup)
+- [Usage](#usage)
+- [Contributing](#contributing)
+
+## Vision
+
+The Cuddly Team believes that pets are not just animals, but that pets are family. We want to help spread the joy that pets bring to our lives and facilitate trusted connections to improve pet care. So we made a social media application that allows pet fans to share experiences and expertise, and grow relationships.
```

Comment:
Suggest to mention all types of users, like the following:

So we made a social media application to gather pets parents, pet fans, pet caretakers and pet service providers, empower the users to share experiences and expertise, and grow relationships.

Reply:
i will add it in the features !